go.mod

```diff
@@ -3,18 +3,17 @@ module github.com/stackup-wallet/stackup-bundler
 go 1.20
 
 require (
-	github.com/blndgs/model v0.18.0
+	github.com/blndgs/model v0.18.1
 	github.com/deckarep/golang-set/v2 v2.3.0
 	github.com/dgraph-io/badger/v3 v3.2103.5
-	github.com/ethereum/go-ethereum v1.11.5
+	github.com/ethereum/go-ethereum v1.13.14
```

Comment:
can't upgrade from Geth 1.11.x I think. Have you tested these changes?

Reply:
fixed